src/main/java/org/apache/mesos/hdfs/executor/AbstractNodeExecutor.java

```diff
@@ -155,7 +155,7 @@ private void addBinaryToPath(String hdfsBinaryPath) throws IOException {
     BufferedWriter bufferedWriter = new BufferedWriter(fileWriter);
     bufferedWriter.write(scriptContent);
     bufferedWriter.close();
-    Runtime.getRuntime().exec("chmod a+x " + pathEnvVarLocation);
+    new ProcessBuilder("chmod", "a+x", pathEnvVarLocation).start();
```

Comment:
In both cases it will create this intermediary object. Method chaining is poor java style so let's put this on two lines.